CHANGELOG.md

```diff
@@ -443,7 +443,7 @@
 - [#1116](https://github.com/request/request/pull/1116) Fix typo in `noProxyHost` definition (@FredKSchott)
 - [#1114](https://github.com/request/request/pull/1114) linting fix: Added a `new` operator that was missing when creating and throwing a new error (@FredKSchott)
 - [#1096](https://github.com/request/request/pull/1096) No_proxy support (@samcday)
-- [#1107](https://github.com/request/request/pull/1107) linting-fix: remove unused variables (@FredKSchott)
+- [#1107](https://github.com/request/request/pull/1107) linting-fix: remove unused letiables (@FredKSchott)
```

Comment:
It looks like you performed a search replace on var with let, and it went a bit too far afield.
There are quite a few of these in the changelog and readme, and probably other places too, they'll need changed back.